gufe/settings/models.py

```diff
@@ -109,40 +103,40 @@ class ThermoSettings(SettingsBaseModel):
        possible.
     """
 
-    temperature: FloatQuantity["kelvin"] = Field(None, description="Simulation temperature, default units kelvin")
-    pressure: FloatQuantity["standard_atmosphere"] = Field(
-        None, description="Simulation pressure, default units standard atmosphere (atm)"
-    )
+    temperature: KelvinQuantity | None = Field(None, description="Simulation temperature in kelvin)")
+    pressure: AtmQuantity | None = Field(None, description="Simulation pressure in standard atmosphere (atm)")
```

Comment:
My understanding is that the agreement was that we would be allowing for different compatible units to be ingested, as long as they were united?

If I try to pass 1 * unit.bar this fails.